src/test/java/nl/tudelft/pixelperfect/event/parameter/EventParameterTest.java

```diff
@@ -152,4 +152,28 @@ public void testHashCodeSame() {
     assertThat(object.hashCode(), equalTo(other.hashCode()));
   }
 
+  /**
+   * The toString method should generate a correct String representation of a generic
+   * EventParameter.
+   */
+  @Test
+  public void testToStringGeneric() {
+    objectGeneric
+        .setSummary("The Answer to the Ultimate Question of Life, the Universe, and Everything");
```

Comment:
No, but it makes testing more fun, which is important given the fact that we need to spend a lot of time on this.